.github/workflows/xpu.yml

```diff
@@ -0,0 +1,68 @@
+name: xpu
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+  pull_request:
+    paths:
+      - ".github/workflows/xpu.yml"
+      - "accelerator/xpu_accelerator.py"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  issues: write
+
+
+jobs:
+  unit-tests:
+    runs-on: [self-hosted, intel]
+    container:
+      image: intel/intel-extension-for-pytorch:2.1.20-xpu
+      ports:
+        - 80
+      options: --privileged -it --rm --device /dev/dri:/dev/dri -v /dev/dri/by-path:/dev/dri/by-path --ipc=host --cap-add=ALL
+    
+    steps:
+    - uses: actions/checkout@v3
+    - name: Check container state
+      shell: bash
+      run: |
+        ldd --version
+        python -c "import torch; print('torch:', torch.__version__, torch)"
+        python -c "import torch; import intel_extension_for_pytorch; print('XPU available:', torch.xpu.is_available())"
+
+    - name: Install transformers
+      run: |
+        git clone https://github.com/huggingface/transformers
```

Comment:
Should we use to a specific release of transformer? then upgrade along with its new releases. Just wonder how easily we get broken with the changes on transformer master.

If using the release version, we can simply pip install w/o compilaiton?

Reply:
Habana Gaudi2's CI installing the transformers is exactly the same as this way. I think this should be pretty safe. Also in current test scope we dont need transformers, so I delete it this time.